spec/models/dojo_spec.rb

```diff
@@ -87,4 +101,35 @@
       it { expect(dojo.to_s).to eq "[PRIVADO] #{day_formatted} - bla" }
     end
   end
+
+  describe 'dojo with participant limit' do
+    let(:dojo) { Dojo.new(day: Time.zone.now, local: 'bla', participant_limit: 3) }
+    let(:new_user){ FactoryGirl.create(:user, name: "Malandro") }
+
+    describe '#has_limit?' do
```

Comment:
Em testes assim, de apenas uma linha, dá pra fazer assim:

```ruby
it { expect(dojo.has_limit?).to be(true) }
```

O rspec é esperto o suficiente pra já colocar o "should be true" como descrição nesses casos, e a asserção é auto explicativa.